tests_integration/.mitmproxy/README.md

```diff
@@ -0,0 +1,4 @@
+# mitmproxy confdir
+
+This directory is meant to be used as configuration directory for `mitmproxy`.
+Check the README in [tests_integration](../tests_integration) for more information.
```

Comment:
There's a typo here.

Reply:
fixed in d2afa6f72e499a90a36236e1e5ebeb07e4087a55